### ISSUE TITLE
Test branch 0.9.10

### ISSUE THREAD

**Issue by mattbaumann1** [NUARIG/competitions#526]
Test application, including the new features and fixes in 0.9.10, on staging (not alpha) server.

Front end changes on this branch:
- [x] grant creator request review is inaccessible #527 
- [x] remove draft grants from my reviews #355 
- [x] Update Submission show layout #461

**Comment by jseraf** [comment]
Currently blocked by #525

**Comment by lmkw** [comment]
re #355 , i still see the draft and can score it. when i go to save that score, it tells me the submission is in draft state. i think the behavior is supposed to be that i can't get to the draft at all?

**Comment by mattbaumann1** [comment]
This ticket was for grants in draft state. I have opened a new ticket for submissions in draft state, #535 .

**Comment by lmkw** [comment]
My fault, ignore my comment above about submissions in draft, Matt opened a separate ticket for that. However, I can still see a review for a grant that is in draft state.
